### PR TITLE
Add the error code to assistant errors when available

### PR DESCRIPTION
**Pull request by aubin-tchoi**
## Description

- Add the error code to assistant errors when available.

## Risk

Low

## Deploy Plan

- Deploy front

### CODE REVIEW

File: front/lib/api/assistant/agent.ts [hunk 1]
```diff
@@ -595,7 +595,7 @@ async function* runMultiActionsAgent(
         messageId: agentMessage.sId,
         error: {
           code: "multi_actions_error",
-          message: `Error running assistant: ${event.content.message}`,
+          message: `Error running assistant: [${event.content.code}] ${event.content.message}`,
```
Comment (spolu) [inline front/lib/api/assistant/agent.ts:598]: mmmmmmmm not sure this will give us the 429 vs 529 will it?

Comment (aubin-tchoi) [reply front/lib/api/assistant/agent.ts:598]: oh i misread event as error

Comment (aubin-tchoi) [reply front/lib/api/assistant/agent.ts:598]: nvm then, not sure we can do much if the code is not in the message

Comment (spolu) [reply front/lib/api/assistant/agent.ts:598]: Yep is has to be done in core 👍

**Pull request closed without merging**